update commands to allow renaming a containerized server (#3690)
uyuni-project/uyuni-docs: modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc

  Be aware that this script refreshes the pillar data for all {salt} clients: the time it takes to run depends on the number of registered clients.
. *Skip this step if the clients are managed via a {productname} proxy.*
  Re-configure the clients directly managed to make them aware of the new hostname and IP address.
  In the {salt} client configuration file, you must specify the name of the new {salt} master ({productname} Server).
  The filename is [path]``/etc/venv-salt-bundle/minion`` or, if you do not use the {salt} bundle, [path]``/etc/salt/minion``:
+
----
master: <new_hostname>
----

. Restart the server.
  On the container host, execute the restart command:
+
----
mgradm restart
----

. To fully propagate the hostname to the {salt} client configuration apply the high state.
  Applying the high state will update the hostname in the repository URLs.

== Proxy Reconfiguration

Any proxy must be reconfigured.
The new server certificate and key must be copied to the proxy.
For more information, see xref:installation-and-upgrade:install-proxy.adoc[].

[NOTE]
====
If you use PXE boot through a proxy, you must check the configuration settings of the proxy.

> Only for non-containerized proxies version 4.3, run the [command]``configure-tftpsync.sh`` setup script and enter the requested information.
> In such a case, on the container host, execute:
> 
> ----
> mgrctl exec -ti -- configure-tftpsync.sh
> ----

Reviewer: This is confusing to me. How about making this `IMPORTANT` instead of `NOTE` and

```suggestion
If you use PXE boot through Non-containerized {productname} Proxy 4.3 [literal]``tftpsync`` needs to be reconfigured.

On the container host, execute:
----
mgrctl exec -ti -- configure-tftpsync.sh
----
```

. On the container host, from the command line, execute the following command to configure the public IP address of the server:
+
----
mgrctl exec -ti -- spacewalk-hostname-rename <PUBLIC_IP_ADDRESS>
----
+
If the server is not using the new hostname, the script fails.
  Be aware that this script refreshes the pillar data for all {salt} clients: the time it takes to run depends on the number of registered clients.
. *Skip this step if the clients are managed via a {productname} proxy.*
  Re-configure the clients directly managed to make them aware of the new hostname and IP address.
  In the {salt} client configuration file, you must specify the name of the new {salt} master ({productname} Server).
  The filename is [path]``/etc/venv-salt-bundle/minion`` or, if you do not use the {salt} bundle, [path]``/etc/salt/minion``:
+
----
master: <new_hostname>
----

. Restart the server.
  On the container host, execute the restart command:
+
----
mgradm restart
----

. To fully propagate the hostname to the {salt} client configuration apply the high state.
  Applying the high state will update the hostname in the repository URLs.

== Reconfigure Proxy

> Any proxy must be reconfigured.

Reviewer 2: ```suggestion
Every proxy must be reconfigured.
```

The new server certificate and key must be copied to the proxy.
For more information, see xref:installation-and-upgrade:install-proxy.adoc[].

[IMPORTANT]
====
If you use PXE boot through a proxy, you must check the configuration settings of the proxy.

> If you use PXE boot through Non-containerized {productname} Proxy 4.3 [systemitem]``tftpsync`` needs to be reconfigured.

Reviewer 2: ```suggestion
If you use PXE boot through non-containerized {productname} Proxy 4.3,[systemitem]``tftpsync`` needs to be reconfigured.
```
